### PR TITLE
Fix Avro CVE

### PR DESCRIPTION
**Pull request by srishti-saraswat**
## Problem
https://confluentinc.atlassian.net/browse/CC-22818
https://confluentinc.atlassian.net/browse/SECOPS-15591 

## Solution
Update common version and remove unused pinned avro version.
Fix schema for transaction.avro

##### Does this solution apply anywhere else?
- [ ] yes
- [ ] no

## Test Strategy
```
mvn dependency:tree | grep org.apache.avro       
[INFO] |  +- org.apache.avro:avro:jar:1.11.3:compile
```

##### Testing done:
- [ ] Unit tests
- [ ] Integration tests
- [ ] System tests
- [ ] Manual tests

### CODE REVIEW

**Comment by srishti-saraswat** [comment]
Build - https://jenkins.confluent.io/job/Confluent%20Public%20Repo%20PR%20builder/job/kafka-connect-datagen/view/change-requests/job/PR-136/4/

**Review by davetroiano — APPROVED** [review]
LGTM... I have one suggestion

**Comment by srishti-saraswat** [comment]
Build - https://jenkins.confluent.io/job/Confluent%20Public%20Repo%20PR%20builder/job/kafka-connect-datagen/view/change-requests/job/PR-136/5/

File: src/main/resources/transactions.avro [hunk 1]
```diff
@@ -43,4 +43,3 @@
                 }}
             ]
          }
-}
```
[the remark this answers is not in the document]
Comment (srishti-saraswat) [reply src/main/resources/transactions.avro:46]: The tests were failing when avro was updated to `1.11.3`. Weirdly they were not failing with older avro versions.
Yeah, I just crosschecked with transactions template, it is working. Let me verify the cloud version and if this bracket error is there in that version or not

**Pull request merged**